sentinel-extension/sentinel-datasource-redis/README.md

```diff
@@ -0,0 +1,37 @@
+# Sentinel DataSource Redis
```

Comment:
Please update and refine the document with pending changes.

Reply:
here i just simply keep same as other data source `README` title.

Reply:
That's right. I'll improve the document later.